README.md

```diff
@@ -41,7 +41,9 @@ To install multiple versions of Node.js, you may find it easier to use a node ve
 
 #### [nodemon](http://nodemon.io/) (optional)
 
-`nodemon` is used to develop [the Component Library](working-on-the-component-library).
+`nodemon` is used to develop [the Component Library][1].
+
+[1]: https://github.com/hmrc/assets-frontend#user-content-component-library
```

Comment:
This isn't how we format links in our README.

It needs to be `[the Component Library](#user-content-component-library)`